examples/12-fluids/03-fluids_isosurface.py

```diff
@@ -0,0 +1,113 @@
+    (-0.0011924505233764648, 1.8596649169921875e-05, 1.125),
+    (-0.2738679385987956, -0.30771426079547065, 0.9112125360807675),
+]
+pl.show_figure(cpos=cpos_whole_mesh, show_axes=True)
+
+###############################################################################
+# Extract the physic variable.
+# ~~~~~~~~~~~~~~~~~~
+# Here we choose to work with the static pressure by default which is a scalar and
+# nodal variable without multi-species/phases. With a multi-species case, we should have
+# select one specific using qualifiers ellipsis pins and connecting a LabelSpace "specie"/"phase".
+
+P_S = dpf.operators.result.static_pressure(streams_container=streams, mesh=whole_mesh).eval()
+print(P_S[0])
+
+pl = DpfPlotter()
+pl.add_field(P_S[0])
+cpos_mesh_variable = [
+    (4.256160478475664, 4.73662111240005, 4.00410065817644),
+    (-0.0011924505233764648, 1.8596649169921875e-05, 1.125),
+    (-0.2738679385987956, -0.30771426079547065, 0.9112125360807675),
+]
+pl.show_figure(cpos=cpos_mesh_variable, show_axes=True)
+
+###############################################################################
+# Evaluate iso-surfaces.
+# ~~~~~~~~~~~~~~~~~~
+# We can finally use the mesh_cut operator on this specific variable.
+# We choose to cut the whole with 5 iso-surface equally spaced between min and max.
+
+max_pressure = 361.8170  # Pa
+min_pressure = -153.5356  # Pa
+number_of_iso_surface = 5
+step = (max_pressure - min_pressure) / number_of_iso_surface
+
+pl = DpfPlotter()
+c_pos_iso = [
+    (4.256160478475664, 4.73662111240005, 4.00410065817644),
+    (-0.0011924505233764648, 1.8596649169921875e-05, 1.125),
+    (-0.2738679385987956, -0.30771426079547065, 0.9112125360807675),
+]
+pl.add_mesh(
+    meshed_region=whole_mesh,
+    style="wireframe",
+    show_edges=True,
+    show_axes=True,
+    color="black",
+    opacity=0.3,
+)
+
+for i in range(number_of_iso_surface):
+    iso_surface = dpf.operators.mesh.mesh_cut(
+        field=P_S[0], iso_value=min_pressure, closed_surface=0, mesh=whole_mesh, slice_surfaces=True
+    ).eval()
+    P_S_step = dpf.Field(location=dpf.locations.overall, nature=dpf.common.natures.scalar)
+    P_S_step.append([min_pressure], i)
+    P_S_step.name = "static pressure"
+    P_S_step.unit = "Pa"
+    pl.add_field(
+        field=P_S_step, meshed_region=iso_surface, style="surface", show_edges=False, show_axes=True
+    )
+    min_pressure += step
+
+pl.show_figure(show_axes=True, cpos=c_pos_iso)
+
+###############################################################################
+# Important note
+# ------------------------------
+# Isosurfaces computation through the `mesh_cut` operator are only supported for Nodal Fields.
+# Thus, for Elemental variables an averaging operation to the Nodes needs to be performed before
+# running the `mesh_cut` operator. This can be done by chaining the `elemental_to_nodal` operator
+# output with the `mesh_cut` operator input.
```

Comment:
```suggestion
# Iso-surfaces computation through the `mesh_cut` operator are only supported for Nodal Fields.
# For Elemental variables, you must perform an averaging operation on the Nodes before
# running the `mesh_cut` operator. This can be done by chaining the `elemental_to_nodal` operator
# output with the `mesh_cut` operator input.
```